### PR TITLE
MCLOUD-2361: OOTB Docker Initialization

### PR DESCRIPTION
**Pull request by bbatsche**
## Purpose of this pull request

This PR modifies the steps for setting up a Magento Cloud Docker based project.

## Affected DevDocs pages

https://devdocs.magento.com/cloud/docker/docker-config.html
https://devdocs.magento.com/cloud/docker/docker-mode-developer.html
https://devdocs.magento.com/cloud/docker/docker-mode-production.html

## Links to Magento source code

https://github.com/magento/magento-cloud-docker/blob/develop/bin/init-docker.sh

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-mode-production.md [hunk 1]
```diff
@@ -18,10 +18,16 @@ To launch the Docker environment in production mode:
 
 1. Add your [Magento access credentials][magento-creds] to the `auth.json` file.
 
-1. Install the template dependencies.
+1. Install the template dependencies and add the default hostname to your `/etc/hosts` file
```
Comment (meker12) [inline src/cloud/docker/docker-mode-production.md:21]:
```suggestion
1. To install the template dependencies and add the default hostname to your `/etc/hosts` file, use the following command to run the Docker initialization script:
```

File: src/cloud/docker/docker-mode-production.md [hunk 2]
```diff
@@ -18,10 +18,16 @@ To launch the Docker environment in production mode:
...
 
    ```bash
-   composer install
+   curl https://raw.githubusercontent.com/magento/magento-cloud-docker/1.1.0/bin/init-docker.sh | bash
+   ```
+
+   If you need to customize some aspect of initializing your environment, see the possible options `init-docker.sh` supports:
```
Comment (meker12) [inline src/cloud/docker/docker-mode-production.md:27]:
```suggestion
   If required, you can add options to the `init-docker.sh` initialization script to customize your Docker environment. Run the following command to see the available options:
```

File: src/cloud/docker/docker-config.md [hunk 3]
```diff
@@ -16,40 +16,15 @@ The `ece-docker build:compose` command overwrites the existing `docker-compose.y
...
-   -  On MacOS and Windows, file synchronization is required for developer mode—use one of the following:
-      -  [docker-sync]
-      -  [mutagen]
+To get started with local development you must have [Docker] installed on your workstation. In addition, macOS and Windows systems require file synchronization between the host and Docker environments. To do this, you should set up either [docker-sync] or [Mutagen].
 
-1. Update the hosts file.
+### Optional Steps
 
-   Before you begin, you must add the following hostname to your `/etc/hosts` file:
+Magento Cloud Docker will bind to port `80` on your host environment. If you have enabled the bundled web server on your workstation you must stop the service before launching the Docker environment.
```
[the remark this answers is not in the document]
Comment (bbatsche) [reply src/cloud/docker/docker-config.md:25]: I mean, I would argue that "bundled" is more accurate than "built-in" but I do see what you mean about the Google results. I guess I'll take my semantic issues up with Apple 😂

Windows and Linux both may also have a web server, but you'd have to do something to install them and we have less certainty about what server it is (even in macOS, the server is only enabled _if you've gone out of your way to enable it_).

So far the only web server Apple has bundled is Apache.

**Pull request merged**